common/dynamicconfig/constants.go

```diff
@@ -484,7 +484,9 @@ Deleted Redirect Rules will be kept in the DB (with DeleteTimestamp). After this
 		"matching.wv.ReachabilityBuildIdVisibilityGracePeriod",
 		3*time.Minute,
 		`ReachabilityBuildIdVisibilityGracePeriod is the time period for which deleted versioning rules are still considered active
-to account for the delay in updating the build id field in visibility.`,
+to account for the delay in updating the build id field in visibility. Not yet supported for GetDeploymentReachability. We recommend waiting 
```

Comment:
do we want to reuse this config which has "buildId" in its name instead of deployment?

Reply:
well.. the search attribute is still called `BuildIds` and this is about how long it takes for the search attribute to update.. but we could make a new one